electron/helpers/utils.js

```diff
@@ -10,8 +11,45 @@ export {
   v4 as generateKey
 } from 'uuid'
 
+const harmfulSwitches = [
+  'remote-debugging-port',
+  'inspect',
+  'inspect-brk',
+  'inspect-brk-node',
+  'inspect-port',
+  'inspect-publish-uid',
+  'js-flags',
+  'proxy-server',
+  'host-rules'
+]
+
+function checkHarmfulSwitchPresence () {
+  for (harmfulSwitch of harmfulSwitches) {
+    if (
+      app.commandLine.hasSwitch(
+        harmfulSwitch
+      )
+    ) {
+      return true
+    }
+  }
+  return false
+}
+
 export const appName = 'muffon'
 
+export const handleHarmfulSwitches = () => {
```

Comment:
@xyloflake This one can be `function`.

Also name can be changed to `checkForHarmfulSwitches`.
`handle` is used for IPC.

Reply:
> @xyloflake This one can be function.
This must NOT be a function (answering why in discord). 
> Also name can be changed to checkForHarmfulSwitches.
How does that make sense? It's handling, not checking. Not changing anything here.

Reply:
@xyloflake You're right.
Bout you still check for them and perform some action.
I guess it can be handling.